### PR TITLE
fix new rubocop 0.30 offenses

### PR DESCRIPTION
**Pull request by abonas**

### CODE REVIEW

File: lib/kubeclient/common.rb [hunk 1]
```diff
@@ -67,8 +67,8 @@ def rest_client
             ssl_client_cert: @ssl_options[:client_cert],
             ssl_client_key: @ssl_options[:client_key]
           }
-          endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' \
-                                                      + @api_version)
+          endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' +
+                                                      @api_version)
```
Comment (simon3z) [inline lib/kubeclient/common.rb]: I think we preferred `"#{@api_endpoint.path}/#{@api_version}"` in other parts.

Same below as well.

Comment (abonas) [inline lib/kubeclient/common.rb]: > I think we preferred "#{@api_endpoint.path}/#{@api_version}" in other parts.
> 
> Same below as well.

right, but it is not related to what this patch fixes.

Comment (simon3z) [inline lib/kubeclient/common.rb]: :+1:

Comment (abonas) [inline lib/kubeclient/common.rb]: > I think we preferred "#{@api_endpoint.path}/#{@api_version}" in other parts.
> 
> Same below as well.

and it causes a "line too long" offence...

**Pull request merged**